Cargo.toml

```diff
@@ -19,3 +19,8 @@ serde = "0.9"
 serde_derive = "0.9"
 serde_json = "0.9"
 tokio-core = "0.1"
+rpassword = "0.4.2"
+
+[features]
+gitm_deny_unknown = []
```

Comment:
As far as i understand, feature names must be globally unique, so I prefixed them with "gitm".